app/models/user.rb

```diff
@@ -13,7 +13,23 @@ def has_role?(role)
   end
 
   def role
-     roles&.first&._type.titleize
+    roles&.first&._type
+  end
+
+  def pending_submissions(user_id)
+    user = User.find(user_id)
+    user.submissions.where(form_id: 2, status: "pending")
+  end
+
+  def total_generator(submissions)
+    sum = 0
+    # submissions.each do |submission|
+    #   submission.data = submission.data.except('name_of_patient', 'relationship_with_employee', 'reporting_manager', 'name_of_patient', 'relationship_with_employee', 'project_name')
+    #   submission.data.values.each do |key, value|
+    #     sum += (key['amount'].to_i)
+    #   end
+    # end
```

Comment:
commented code and you are using this method ? You need to be careful for these mistakes

Reply:
We will modify this function